api/config/config_local.json

```diff
@@ -103,7 +103,9 @@
     "enableCohortBuilderV2": true,
     "enableConceptSetSearchV2": true,
     "enableReportingUploadCron": true,
-    "enableCustomRuntimes": true
+    "enableCustomRuntimes": true,
+    "bufferBillingProjectsInPerimeter": false,
+    "badRequestIsOkWhenAlreadyInPerimeter": true
```

Comment:
true is the more lenient option, so we should keep it true for a while, especially in prod.

has no effect when `bufferBillingProjectsInPerimeter` is true.